pegasus/sites.v3/code.org/public/css/donors.css

```diff
@@ -96,3 +97,99 @@ table td {
 #donorslider .donor-button button {
   margin: 12px 8px;
 }
+
+
+
+/* Styles for the updated Supporters section on the Homepage
+- changed May 2022 */
+.homepage-supporters {
+  padding: 2em 2em 4em;
+  text-align: center;
+}
+.homepage-supporters .donor-button {
+  margin: 0 auto 1.5em;
+}
+
+/* Styles for the updated Supporters page
+- changed May 2022 */
+.clearfix {
+  clear: both;
+}
+
+.no-padding {
+  padding: 0 !important;
```

Comment:
Just curious, why do we need the `!important` on the various properties on this page

Reply:
@jamescodeorg it's to override styles (usually from other external stylesheets) that prevents the elements from looking how we want it to in a particular instance — it's not the best practice, but in cases like this where we don't want any padding, etc… it does the job

Reply:
Do you know which stylesheet(s) the unwanted styles are coming from?

Reply:
In this case they're overriding my own styles — for example on the Supporters page the `.flex-container` class has `padding: 2em` but on the teal header bar where it says "Institutional Supporters " and "Individual Supporters" I wanted to keep that class on there, but just needed to remove the padding.

In general though classes like these can be used to override styles coming from a framework or template css (like Bootstrap) without inline or more specific classes on an element.

Reply:
I see, thanks for the detailed explanation!

Reply:
Yw! Happy to chat about CSS anytime 😄